src/intrinsics/ecma262/Object.js

```diff
@@ -495,11 +495,27 @@ export default function(realm: Realm): NativeFunctionValue {
   });
 
   // ECMA262 19.1.2.9
-  if (!realm.isCompatibleWith(realm.MOBILE_JSC_VERSION) && !realm.isCompatibleWith("mobile"))
-    func.defineNativeMethod("getOwnPropertySymbols", 1, (context, [O]) => {
+  if (!realm.isCompatibleWith(realm.MOBILE_JSC_VERSION) && !realm.isCompatibleWith("mobile")) {
+    let getOwnPropertySymbols = func.defineNativeMethod("getOwnPropertySymbols", 1, (context, [O]) => {
+      if (O instanceof AbstractValue && realm.isInPureScope()) {
+        let obj = O instanceof AbstractObjectValue ? O : To.ToObject(realm, O);
+
```

Comment:
I wonder if we also want to call `realm.callReportObjectGetOwnProperties` here?

Or maybe move the call from here: https://github.com/facebook/prepack/blob/47cb48b438e46d8c353e98618033fef7d9dca560/src/methods/properties.js#L2004 to here https://github.com/facebook/prepack/blob/47cb48b438e46d8c353e98618033fef7d9dca560/src/methods/own.js#L40

@NTillmann thoughts?

Reply:
I can add a `realm.callReportObjectGetOwnProperties` call here. That's a good shout.